include/realizations/catchment/Bmi_Module_Formulation.hpp

```diff
@@ -455,12 +437,10 @@ namespace realization {
         time_t bmi_model_start_time_forcing_offset_s;
         /** A configured mapping of BMI model variable names to standard names for use inside the framework. */
         std::map<std::string, std::string> bmi_var_names_map;
-        /** Whether the backing model uses/reads the forcing file directly for getting input data. */
-        bool bmi_using_forcing_file;
-        std::string forcing_file_path;
         bool model_initialized = false;
 
         std::vector<std::string> OPTIONAL_PARAMETERS = {
+                BMI_REALIZATION_CFG_PARAM_REQ__USES_FORCINGS
```

Comment:
I should probably either change the name, or comment on why `REQ` remains in it.